collector/src/bin/collector.rs

```diff
@@ -628,6 +629,18 @@ enum Commands {
         #[command(flatten)]
         db: DbOption,
     },
+
+    /// Displays diff between two local bench result.
+    BenchCmp {
+        #[command(flatten)]
+        db: DbOption,
+
+        #[arg(long)]
```

Comment:
Could you please add doc comments to `base` and `modified`? So that it shows in the CLI help.